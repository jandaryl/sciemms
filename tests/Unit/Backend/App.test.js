import { mount } from '@vue/test-utils'
import App from '@/App.vue'

descriv
