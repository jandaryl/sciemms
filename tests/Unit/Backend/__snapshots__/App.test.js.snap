// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App is matched in the previos snapshots 1`] = `<router-view-stub />`;
